user_research/reset_env.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: clean-and-migrate-user-research-env
+spec:
+  template:
+    spec:
+      securityContext:
+        runAsUser: 100
+      containers:
+        - name: clean-and-migrate-user-research-env
+          image: flyway/flyway:7.7.0-alpine
```

Comment:
_observation_
It looks like if we bundle flyway into our `quay.io/hmpps/hmpps-interventions-service` image here (and maybe use repeatable migrations?), we probably won't need a separate deployment to apply the changes